cmake/modules/FindTBB.cmake

```diff
@@ -197,7 +197,12 @@ if(NOT TBB_FOUND)
   ##################################
 
   if(TBB_INCLUDE_DIRS)
-    file(READ "${TBB_INCLUDE_DIRS}/tbb/tbb_stddef.h" _tbb_version_file)
+    # Use new oneTBB version header if it exists.
+    if(EXISTS "${TBB_INCLUDE_DIRS}/tbb/version.h")
```

Comment:
Apart from the copyright notice in tbb/version.h there is only
```
#include "../oneapi/tbb/version.h"
```
https://github.com/oneapi-src/oneTBB/blob/96af5d3621c766d7b0e55d6c8c93738f68cd8ead/include/tbb/version.h#L17C1-L17C35
So should oneapi/tbb/version.h not be being used instead?